File: src/mem/mySchedule/inter_queue.cc

#include "inter_queue.hh"
#include "mem/mem_interface.hh"

//#include "debug/interQ.hh"

namespace gem5
{


namespace memory{



///interQueue
InterQueue::InterQueue(const InterQueueParams& p) : 
mctrl(p.mc),
sys(p.system),
SimObject(p)
{}



}

} 

File: src/mem/mySchedule/InterQueue.py
from m5.params import *
from m5.proxy import *
from m5.SimObject import SimObject


class InterQueue(SimObject):
    type       = 'InterQueue'
    abstract   = True
    cxx_header = 'mem/mySchedule/inter_queue.hh'
    cxx_class  = 'gem5::memory::InterQueue'

    mc         = Param.MemCtrl(NULL, "memory ctrl owner")
    system     = Param.System(Parent.any, "System that the controller belongs to.")
